source/dicom/valuerep.py

```diff
@@ -48,6 +48,20 @@ class DSfloat(float):
     """
     __slots__ = 'original_string'
 
+    def __getstate__(self):
```

Comment:
I'm still not sure as to: why are these methods added for and why are you keying off of `__slots__`?